internal/cmd/commands/dev/dev.go

```diff
@@ -450,6 +459,24 @@ func (c *Command) Run(args []string) int {
 		c.DevHostId = fmt.Sprintf("%s_%s", static.HostPrefix, c.flagIdSuffix)
 		c.DevTargetId = fmt.Sprintf("%s_%s", tcp.TargetPrefix, c.flagIdSuffix)
 	}
+	if c.flagSecondaryIdSuffix != "" {
+		if len(c.flagSecondaryIdSuffix) != 10 {
+			c.UI.Error("Invalid secondary ID suffix, must be exactly 10 characters")
+			return base.CommandUserError
+		}
+		if !handlers.ValidId(handlers.Id("abc_"+c.flagSecondaryIdSuffix), "abc") {
+			c.UI.Error("Invalid secondary ID suffix, must be in the set A-Za-z0-9")
+			return base.CommandUserError
+		}
+		c.DevSecondaryTargetId = fmt.Sprintf("%s_%s", tcp.TargetPrefix, c.flagSecondaryIdSuffix)
+	}
+
+	if c.flagIdSuffix != "" && c.flagSecondaryIdSuffix != "" {
+		if strings.EqualFold(c.flagIdSuffix, c.flagSecondaryIdSuffix) {
+			c.UI.Error("Primary and secondary ID suffixes are equal, must be distinct")
+			return base.CommandUserError
+		}
+	}
```

Comment:
We could just write this as a single statement, no?
```suggestion
	if c.flagIdSuffix != "" && c.flagSecondaryIdSuffix != "" &&
		strings.EqualFold(c.flagIdSuffix, c.flagSecondaryIdSuffix) {
		c.UI.Error("Primary and secondary ID suffixes are equal, must be distinct")
		return base.CommandUserError
	}
```

Reply:
Yep - I just thought it read a little clearer with separate statements, addressed in [0c70f05](https://github.com/hashicorp/boundary/pull/2738/commits/0c70f05ddfe65ab9fdb9d9bfd0b19bf5708b0308)